ads_plugin.py

```diff
@@ -731,7 +726,7 @@ def on_compute_morphometrics_button(self, event):
                 wx.LogError("Cannot save current data in file '%s'." % pathname)
 
         # Generate and load the index image
-        original_image_name = (axon_mask_overlay.name).split("-axon")[0]
+        original_image_name = (axon_mask_overlay.name).split("-axon-manual_pred")[0]
```

Comment:
```suggestion
        original_image_name = (axon_mask_overlay.name).split("-axon")[0]
```
Same as above (internal file/overlay naming I think)